README.md

```diff
@@ -69,7 +79,7 @@ $ kubectl create ns open-cluster-management-agent-addon
 namespace/open-cluster-management-agent-addon created
 ```
 
-Deploy the subscription add-on on the _hub_ cluster's _managed_ cluster namespace. For example, `cluster1`.
+Deploy the subscription add-on on the _hub_ cluster. For the value of `MANAGED_CLUSTER_NAME`, use the `<cluster name>` that you used for `clusteradm join --cluster-name <cluster name>`.
```

Comment:
It might be easier for readers if we say ... chose managed cluster names from `kubectl get managedclusters`.

Reply:
Done as suggested @rokej please have another look. 

/unhold

Reply:
Thank you @mikeshng